packages/odata-v2/src/properties-util.spec.ts

```diff
@@ -1,11 +1,36 @@
-import { testService } from '@sap-cloud-sdk/test-services-odata-v2/test-service';
-import { isNavigationProperty } from '@sap-cloud-sdk/odata-common/dist/properties-util';
+import {
+  TestEntity,
+  testService
+} from '@sap-cloud-sdk/test-services-odata-v2/test-service';
+import {
+  nonEnumerable,
+  isNavigationProperty
+} from '@sap-cloud-sdk/odata-common/dist/properties-util';
+const { testEntityApi } = testService();
 
 describe('properties-util', () => {
-  it('returns true for one to many navigation property', () => {
-    const { testEntityApi } = testService();
+  it('returns false for non-enumerable property', () => {
+    nonEnumerable(TestEntity, '_entityName');
+    expect(
+      Object.getOwnPropertyDescriptor(TestEntity, '_entityName')?.enumerable
+    ).toBeFalsy();
+  });
+
+  it('returns true for one to one navigation property ', () => {
+    expect(isNavigationProperty('toSingleLink', testEntityApi.schema)).toBe(
+      true
+    );
+  });
+
+  it('returns true for one to many navigation property ', () => {
     expect(isNavigationProperty('toMultiLink', testEntityApi.schema)).toBe(
       true
     );
   });
+
+  it('returns false for string property', () => {
+    expect(isNavigationProperty('stringProperty', testEntityApi.schema)).toBe(
```

Comment:
I guess this is already tested via the common package?

Reply:
I removed this test